modules/packages/apt_get

```diff
@@ -1,10 +1,23 @@
-#!/usr/bin/python
+#!/bin/sh
+"true" '''\'
+PYTHON_BINS="/usr/bin/python /usr/bin/python3"
```

Comment:
In the interest of being future oriented, I would reverse the order of these, so we prefer python3 over python ("python" is always python2, AFAIK).